src/lib/decodeData.ts

```diff
@@ -85,12 +86,13 @@ export const isValidTuple = (valueType: string, valueContent: string) => {
       );
     }
 
+    const valueTypeBytesLength = valueTypeParts[i].slice(5);
```

Comment:
Where is the 5 coming from?

Reply:
The 5th index character in `bytes5` for instance, to extract the `5`.
I have refactored to: 

```js
    const valueTypeBytesLength = valueTypeParts[i].split('bytes')[1];
```